web-app/.gitignore

```diff
@@ -4,3 +4,4 @@
 yarn-error.log
 /cypress/screenshots/
 /__sapper__/
+/static/tailwind.css
```

Comment:
I added it to `postinstall` so that it should always be available, but I didn't really want to commit it yet because it's sort of intended to be a temporary solution until I get to around to putting in the webpack loader for postcss